features/admin.organizations.v1/constants/organization-constants.ts

```diff
@@ -64,6 +65,29 @@ export class OrganizationManagementConstants {
         "cdaefcee-ecdb-47af-8538-174ec13292db"
     );
 
+    public static readonly ORGANIZATION_AUTHENTICATOR: GenericAuthenticatorInterface = {
+        authenticators: [
+            {
+                authenticatorId: "T3JnYW5pemF0aW9uQXV0aGVudGljYXRvcg",
```

Comment:
Shall we take these IDs from defined constants that are already defined in AuthenticatorManagementConstants?

Please check and update the other places as well.

```suggestion
                authenticatorId: AuthenticatorManagementConstants.ORGANIZATION_ENTERPRISE_AUTHENTICATOR_ID,
```